bedrock/settings/__init__.py

```diff
@@ -18,9 +18,7 @@
     try:
         from .local import *  # noqa
     except ImportError as exc:
-        exc.args = tuple(['%s (did you rename bedrock/settings/local.py-dist?)' %
-                          exc.args[0]])
-        raise exc
+        'local.py is supported, but no longer necessary'
```

Comment:
Ideally not having a local.py would become the norm rather than the exception, so no, I wouldn't see the point of printing a warning here. Once we're farther along and have refactored the local settings into environment variables on all the servers though, it probably makes sense to add a deprecation warning if you *have* a local.py though.